datasets/standvastgoed/vastgoed_0.0.4.json

```diff
@@ -0,0 +1,477 @@
+{
+  "type": "dataset",
+  "id": "standVastgoed",
+  "version": "0.0.4",
+  "status": "beschikbaar",
+  "auth": "FP/MDW",
+  "title": "stand van de vastgoedgegevens",
+  "description": "stand op één bepaalde peildatum van gegevens in het vastgoed domein",
+  "publisher": "Onderzoek en Statistiek, team Basisstatistiek",
+  "contactPoint" : {
+    "vcard:fn" : "Basisstatistiek",
+    "vcard:hasEmail" : "basisstatistiek.ois@amsterdam.nl"
+  },
```

Comment:
Excuses, de spec heeft het weliswaar over vCard, maar het meta schema (en onze code die daar iets mee doet) zegt dat `contactPoint` er zo uit moet zien in dit geval:
```json
  "contactPoint" : {
    "name" : "Basisstatistiek",
    "email" : "basisstatistiek.ois@amsterdam.nl"
  },
```

Ik zal een issue aanmaken op onze backlog op de spec aan te passen.